smorest_crud/view.py

```diff
@@ -164,9 +166,15 @@ class ResourceView(CRUDView):
     delete_enabled: bool = False
 
     def _lookup(self, pk):
-        """Get model by primary key."""
-        item = self.model.query.get_or_404(pk)
-        return item
+        """Get model by primary key or extid."""
+        if isinstance(pk, int) or str.isdigit(pk):
+            item = self.model.query.get_or_404(pk)
+            return item
+        else:
+            item = self.model.query.filter_by(extid=pk).one_or_none()
```

Comment:
This code is highly-specific to JetKit. `extid` is not a column any other users will have. This isn't suitable for general usage.
Perhaps we should start making our primary keys be UUIDs, or subclass and override this lookup in JetKit.

Reply:
Oh, that's a blunder on my side.

Is creating a reusable decorator for the `extid` lookup in JetKit a good idea?

Reply:
Yeah, not sure it needs to be a decorator. We could subclass collection and resource view in JetKit and override `lookup`

Reply:
Roger that